src/style.scss

```diff
@@ -0,0 +1,137 @@
+$color-blue: lightskyblue;
+$color-coral: lightcoral;
+$color-yellow: lightgoldenrodyellow;
+$color-b: black;
+$color-grey: lightgray;
+
+$fontsize-L: 26px;
+$fontsize-M: 16px;
+$fontsize-XL: 34px;
+
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  display: flex;
+  gap: 50px;
+  flex-direction: column;
+
+  h1 {
+    font-size: $fontsize-XL;
```

Comment:
- [ ] Please make sure to fix the below style issues to match the wireframe design.
  - Please make sure the Leaderboard heading is aligned to the left instead of the center.
  - Please make sure to add a border around the scores section.
  - Please make sure the Submit button is aligned to the right instead of the center.

### Your design

![image](https://github.com/vangelif/LB/assets/83803414/04cc0d76-fff4-40a5-ba33-fca09dd87fd6)

### Wireframe design

![image](https://github.com/vangelif/LB/assets/83803414/3888e615-7a91-492b-bb9d-2a4dcec754bd)